staging/src/k8s.io/kubectl/pkg/generate/versioned/secret_for_tls.go

```diff
@@ -121,12 +146,34 @@ func readFile(file string) ([]byte, error) {
 	return b, nil
 }
 
+// validateCABundle checks that the given bytes contain at least one PEM-formatted certificate.
```

Comment:
If this is the intermediates, don't we need to check that they form a chain up from the cert itself?

Reply:
Yeh, in principle we could do this, but I'm wary of second-guessing exactly how organizations build certificate hierarchies. I was afraid of the risk of having to tweak this logic over multiple releases :)